pkg/trafficrouting/manager.go

```diff
@@ -274,6 +275,121 @@ func (m *Manager) FinalisingTrafficRouting(c *TrafficRoutingContext, onlyRestore
 	return true, nil
 }
 
+// RestoreGateway restore gateway resources without graceful time
+// bool return value indicates whether the gateway resources has been successfully restored
+// error return value indicates error encountered during restoring
+func (m *Manager) RestoreGateway(c *TrafficRoutingContext) (bool, error) {
```

Comment:
the first return variable seems redundant, consider remove it. actually this problem also applies to other func changed in this file

Reply:
This is intentional, for two reasons:

Reply:
it is mainly to maintain consistency with other traffic routing functions.
anyway, i removed the first return variable.